chromium_src/chrome/browser/ui/ui_features.h

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_UI_FEATURES_H_
+#define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_UI_FEATURES_H_
+
+#include "src/chrome/browser/ui/ui_features.h"  // IWYU pragma: export
+
+#include "base/feature_list.h"
+
+namespace features {
+
+BASE_DECLARE_FEATURE(kBraveNtpSearchWidget);
```

Comment:
I don't see why you need to put this feature into `chromium_src` override.

you can add `brave/browser/ui/ui_features.h(cc)` and put it there. If there will be compilation or link issues, use `brave_ui_features.h(cc)` name.